data_import/models.py

```diff
@@ -117,18 +124,23 @@ def file_url_as_attachment(self):
         """
         return self.file.storage.url(self.file.name)
 
-    @property
```

Comment:
It's funny, I wonder if this was intended to be used in templates (the way `download_url` is! But I don't think it ever was. Removing this seems to be fine.

Reply:
Yeah, I grepped for it and only found it in two places, here in the model and in the serializer.  I'm certain it's safe to remove.